packages/grid/_modules_/grid/hooks/features/rows/useGridEditRows.ts

```diff
@@ -201,10 +223,11 @@ export function useGridEditRows(apiRef: GridApiRef) {
   );
 
   const handleExitEdit = React.useCallback(
-    (params: GridCellParams, event: React.SyntheticEvent) => {
```

Comment:
Are the `params` no longer of the given type?

Reply:
Fixed.